sixtyfps_compiler/expression_tree.rs

```diff
@@ -55,6 +56,9 @@ impl BuiltinFunction {
             BuiltinFunction::Debug => {
                 Type::Function { return_type: Box::new(Type::Void), args: vec![Type::String] }
             }
+            BuiltinFunction::Format => {
+                Type::Function { return_type: Box::new(Type::String), args: vec![Type::String, Type::String] }
```

Comment:
The best would be to have support for variadic function.  I believe the `debug`  function would also gain in being variadic.  
Either a new bolean `is_variadic` in `Type::Function`   or a new enum variant in Type, such as `Type::VariadicParameter(Box<Type>)`